### PR TITLE
More SCA checks

### PR DESCRIPTION
**Pull request by markgov**
More Errors found in SCA added to call details and added ignores to the alert

### CODE REVIEW

**Review by davidkelliott — COMMENTED** [review]
You can remove the comment about resolving later, this is fine as we can't restrict access to resources any further because this possibly is applied to all accounts and we don't know what the resources will be

File: terraform/modules/iam_baseline/main.tf [hunk 1]
```diff
@@ -8,9 +8,12 @@ resource "aws_iam_user" "cicd_member_user" {
 resource "aws_iam_group" "cicd_member_group" {
   name = "cicd-member-group"
 }
-
+#Adding checkov skips into bellow but this will probably need to be looked at in a later story#
```
Comment (ewastempel) [inline terraform/modules/iam_baseline/main.tf:11]: have you already created a follow on story to look at it then?
I am worried we will skip them checks and then never look at it; so I would say, if you don’t think it’s resolved, don’t skip them

Comment (markgov) [reply terraform/modules/iam_baseline/main.tf]: removed line 11  as we can't restrict access to resources

**Review by ewastempel — COMMENTED** [review]
LGTM

**Review by davidkelliott — APPROVED** [review]

**Pull request merged**